### PR TITLE
Card magnification

### PR DESCRIPTION
**Pull request by ahaurw01**
It's a start. Let me know what you think. If you press 'z' and mouse over cards on the battlefield, a magnification lens thing shows over it. This doesn't yet work for tapped cards or cards in the left panel.

### CODE REVIEW

File: app/components/game-card.js [hunk 1]
```diff
@@ -30,6 +36,30 @@ export default Ember.Component.extend({
   /** @property {Boolean} only interact with the cards if you're player one */
   readOnly: false,
 
+  /** Attach document key handlers for magnifying purposes. */
+  attachKeyHandlers: Ember.on('init', function () {
```
Comment (SirZach) [inline app/components/game-card.js]: haven't tested this yet but do you think implementing the keyDown and keyUp actions directly on the component would prevent you from having to inspect whether or not you clicked down on the game-card?

File: app/components/game-card.js [hunk 2]
```diff
@@ -56,5 +86,63 @@ export default Ember.Component.extend({
     if (this.get('canTap') && !this.get('readOnly')) {
       this.sendAction('tap', this.get('gameCard'));
     }
+  },
+
+  didInsertElement() {
+    // Calculate the bounds of the image inside this element.
+    var $img = $(this.element).find('img');
+    this.set('magnifyMinX', $img.offset().left);
```
Comment (SirZach) [inline app/components/game-card.js]: setProperties could help here and do all the sets in one run loop

File: app/components/game-card.js (hunk 1 above)
Comment (ahaurw01) [inline app/components/game-card.js]: These guys didn't seem to interfere at all with the click handlers.

Comment (SirZach) [inline app/components/game-card.js]: sorry didn't mean to say whether or not you clicked down on the game-card but if you keyed down or keyed up on the game-card

Comment (ahaurw01) [inline app/components/game-card.js]: Not too sure. An alternative approach could set one set of key listeners on the game controller and bind a property to all game cards.

**Comment by SirZach** [comment]
works amazingly. thanks man!

**Comment by ahaurw01** [comment]
Thanks! I'll address the setProperties in another PR. Also, there's something weird about the left column - it seems to break all magnification when it is open. I'll check it out later.

**Pull request merged**